decompiler/structures/pointers.py

```diff
@@ -72,6 +73,8 @@ def _initialize_points_to(self, cfg: ControlFlowGraph):
             if self._assigns_to_address_of(instr):
                 if not isinstance(instr.value.operand, BinaryOperation):
                     self.points_to[instr.destination].add(instr.value.operand.name)
+            if isinstance(instr, Assignment) and isinstance(instr.value, GlobalVariable) and isinstance(instr.value.type, Pointer):
+                self.points_to[instr.destination] = set()
```

Comment:
Please elaborate on this new functionality and reflect on it in the doc string. Kind of seems like nothing can point on a assignment using a global Variable as value (but not really, since it depends on the iteration of the instructions if this really happens)

Reply:
Will remove. I think the idea was to track the variables that are copies of a global variables of type pointer. But propagation of global variables should be handled properly in another issue, consider it as a debug-trying-to-fix-an-error-leftover.

Reply:
Removed.